src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -0,0 +1,89 @@
+<Project>
+
+  <PropertyGroup>
+    <OutputInRepoRoot>true</OutputInRepoRoot>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="..\Microsoft.AspNetCore.App.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion)</TargetFramework>
+    <IsShippingPackage>true</IsShippingPackage>
+
+    <PackageDescription>This package provides reference assemblies for compiling an ASP.NET Core application. This package is an internal implementation of the .NET Core SDK and is not meant to be used as a normal PackageReference.</PackageDescription>
```

Comment:
Maybe this should also mention documentation and other design-time assets?

For comparison, this is what I have currently for Microsoft.NETCore.App.Ref:

> A set of .NET APIs that are included in the default .NET Core application model. Contains reference assemblies, documentation, and other design-time assets.

The first sentence is Microsoft.NETCore.App's existing description, second sentence is based on the design doc definition, "reference assemblies, documentation, and other design-time assets".

I see I'm missing the internal implementation warning--filed https://github.com/dotnet/core-setup/issues/5107 to add that.